CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ endif()
 
 project(Imath VERSION 3.1.0 LANGUAGES C CXX)
 
-set(IMATH_VERSION_EXTRA "dev" CACHE STRING "Extra version tag string for Imath build")
+set(IMATH_VERSION_SUFFIX "-dev" CACHE STRING "Extra version tag string for Imath build, such as -dev, -beta1, etc.")
```

Comment:
OIIO calls the VERSION_RELEASE_TYPE, I'd suggest using that in the hopes of encouraging a mini-convention. No reason to be different here.

Reply:
Sure, that's fine.

BTW, Since this only changes a few lines, and directly conflicts with your other PR, I would also be happy for you to just fold these changes (or the equivalent of your choice) directly into your other PR by hand and we can abandon this one. That might be easier than having a merge conflict.